### DISCUSSION TITLE
Homebrew tap/formula for installing image

### DISCUSSION THREAD

**Discussion by Bsivakumar29** [Homebrew/discussions#2448] [Everyday usage]
We are working on creating a new homebrew tap/formula for installing generic image based on the below StackExchange write-up.  It is like a configuring dual boot on Mac. 

We have planned to write the shell script files for unattended installation and place those script files in homebrew’s Git hub repositories. Once the homebrew tap/formula is created, we install the Ubuntu image directly from the homebrew package manager. 

As a part above, we were currently working on creating an unattended installation image. Is that possible to install the image using the homebrew package manager? 

https://apple.stackexchange.com/questions/312264/installing-ubuntu-without-dvd-or-flash-drive

**Comment by SMillerDev** [comment]
Homebrew doesn't generally write outside it's standard install location, so if it's possible it'll be very difficult
